dotnet/TF2SA.Web/Program.cs

```diff
@@ -8,7 +8,7 @@
 
 // Add services to the container.
 builder.Services.AddControllersWithViews();
-builder.Services.AddMariaDb(builder.Configuration);
+builder.Services.AddDataLayer(builder.Configuration);
 builder.Services.AddScoped<IPlayersRepository<Player, ulong>, PlayersRepository>();

```

Comment:
Please move this method for adding the repository to the `TF2SA.Data/DependencyInjection.cs`.
We should register all data-related dependencies to that file, as it is with the dbcontext.